instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/internal/instrumentation/logging/LogbackAppenderInstaller.java

```diff
@@ -19,7 +19,19 @@
 class LogbackAppenderInstaller {
 
   static void install(ApplicationEnvironmentPreparedEvent applicationEnvironmentPreparedEvent) {
-    Optional<OpenTelemetryAppender> existingOpenTelemetryAppender = findOpenTelemetryAppender();
+    Optional<io.opentelemetry.instrumentation.logback.mdc.v1_0.OpenTelemetryAppender>
+        existingMdcAppender =
+            findAppender(
+                io.opentelemetry.instrumentation.logback.mdc.v1_0.OpenTelemetryAppender.class);
+    if (existingMdcAppender.isPresent()) {
+      initializeMdcAppenderFromProperties(
+          applicationEnvironmentPreparedEvent, existingMdcAppender.get());
+    } else if (isLogbackMdcAppenderAddable(applicationEnvironmentPreparedEvent)) {
+      addMdcAppender(applicationEnvironmentPreparedEvent);
```

Comment:
This case does not seem to be tested? 

With the OpenTelemetry Logback appender, it is tested in `smoke-tests-otel-starter` without a Logback XML file (assertions: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fe4de02df680bf233f98646bcf97bbf66da7cd10/smoke-tests-otel-starter/spring-boot-common/src/main/java/io/opentelemetry/spring/smoketest/AbstractOtelSpringStarterSmokeTest.java#L204).

Reply:
test added